packages/sources/crypto-api-v2/README.md

```diff
@@ -0,0 +1,250 @@
+# Chainlink CryptoAPIs V2 External Adapter
+
+### Environment Variables
+
+| Required? |    Name     |                             Description                             | Options | Defaults to |
+| :-------: | :---------: | :-----------------------------------------------------------------: | :-----: | :---------: |
+|    ✅     |   API_KEY   | An API key that can be obtained from [here](https://my.cryptoapis.io/register) |         |             |
+|           | API_TIMEOUT |                          Timeout parameter                          |         |   `30000`   |
```

Comment:
Is the default here 3000 or 30000?
Since this is an API key for all adapters, I think we don't have to include it here.

Reply:
It's 30000.  I just copied this from the CryptoAPI's ReadMe.  Is that ReadMe incorrect?

Reply:
Ah it looks like that's correct